contrib/nydusify/pkg/backend/oss.go

```diff
@@ -23,16 +24,26 @@ import (
 )
 
 const (
-	splitPartsCount = 4
-	// Blob size bigger than 100MB, apply multiparts upload.
-	multipartsUploadThreshold = 100 * 1024 * 1024
+	// We always use multipart upload for OSS, and limit the
+	// multipart chunk size to 500MB.
+	multipartChunkSize = 500 * 1024 * 1024
```

Comment:
Why is the multipart chunk size increased? 500MB seems pretty large for a single upload stream.

Reply:
Updated comment, for multipart uploads, OSS has a maximum number of 10000 chunks, so we can only upload blob sizes of about 1T (10000 * 100MB) if we use `100MB` chunk, maybe adjusting to `200MB`? Anyway, it's ok for me.